.env.example

```
@@ -7,7 +7,15 @@ NODE_ENV=production
 VITE_GOOGLE_CLIENT_ID=your_google_client_id_here.apps.googleusercontent.com
 VITE_GOOGLE_API_KEY=your_google_api_key_here
 
-# Configuration Google Calendar (optionnel)
+# Configuration Google Calendar (Service Account)
+# Pour activer la synchronisation avec Google Calendar
+# Email: planningadmin@apaddicto.iam.gserviceaccount.com
+# ID unique: 117226736084884112171
+GOOGLE_SERVICE_ACCOUNT_EMAIL=planningadmin@apaddicto.iam.gserviceaccount.com
+GOOGLE_CALENDAR_API_KEY=d1a5eccfbb5d5a35fbbe564b1dd5b914ec2a4939
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_CALENDAR_API_KEY key should go before the GOOGLE_SERVICE_ACCOUNT_EMAIL key

(UnorderedKey)